src/test/java/com/hocs/test/glue/CreateCaseStepDefs.java

```diff
@@ -54,7 +54,7 @@ public void iAmPresentedWith(String userView) {
         }
     }
 
-    @Given("^I create a single case \"([^\"]*)\"$")
+    @Given("^The current user creates a single case \"([^\"]*)\"$")
```

Comment:
the should be lower case here to help with readability when writing cucumber steps.

Reply:
maybe better to change this to the following:
```suggestion
    @Given("^the current user creates a single \"([^\"]*)\" case$")
```

Reply:
capitalised sentences resolved, Eamon is in the middle of work so will leave the 2nd piece to later to avoid clashes.

Reply:
That's fine, just make sure he makes a note to go back and do them.